File: app/src/main/java/com/xiaweizi/qnews/activity/MyApplication.java
package com.xiaweizi.qnews.activity;

import android.app.Application;

import com.blankj.utilcode.utils.Utils;
import com.zhy.http.okhttp.OkHttpUtils;

import java.util.concurrent.TimeUnit;

import okhttp3.OkHttpClient;

/**
 * 工程名：  QNews
 * 包名：    com.xiaweizi.qnews
 * 类名：    MyApplication
 * 创建者：  夏韦子
 * 创建日期： 2017/2/9
 * 创建时间： 9:55
 */

public class MyApplication extends Application {

    @Override
    public void onCreate() {
        super.onCreate();

        Utils.init(getApplicationContext());
//        Bmob.initialize(this, "5597c24c18ec706d100033f915b79153");


        OkHttpClient okHttpClient = new OkHttpClient.Builder()
//                .addInterceptor(new LoggerInterceptor("TAG"))
                .connectTimeout(10000L, TimeUnit.MILLISECONDS)
                .readTimeout(10000L, TimeUnit.MILLISECONDS)
                //其他配置
                .build();

        OkHttpUtils.initClient(okHttpClient);
    }
}

File: app/src/main/java/com/xiaweizi/qnews/activity/MyApplication.kt
package com.xiaweizi.qnews.activity

import android.app.Application
import cn.jpush.android.api.JPushInterface
import com.blankj.utilcode.utils.Utils
import com.zhy.http.okhttp.OkHttpUtils
import com.zhy.http.okhttp.log.LoggerInterceptor
import okhttp3.OkHttpClient
import java.util.concurrent.TimeUnit

/**
 * class：   QNews
 * author：  xiaweizi
 * date：    2017/9/9 15:23
 * e-mail:   1012126908@qq.com
 * desc:     Application
 */
class MyApplication : Application() {

    override fun onCreate() {
        super.onCreate()

        Utils.init(applicationContext)

        JPushInterface.setDebugMode(false)
        JPushInterface.init(this)

        val okHttpClient = OkHttpClient.Builder()
                .addInterceptor(LoggerInterceptor("TAG"))
                .connectTimeout(10000L, TimeUnit.MILLISECONDS)
                .readTimeout(10000L, TimeUnit.MILLISECONDS).build()

        OkHttpUtils.initClient(okHttpClient)
    }
}